share/setup.nsi

```diff
@@ -1,179 +0,0 @@
-Name "Silk Core (-bit)"
-
-RequestExecutionLevel highest
-SetCompressor /SOLID lzma
-
```

Comment:
These files need to be in the .gitignore rather than deleted don't they? Someone please confirm.